validate/validate_test.go

```diff
@@ -114,6 +114,7 @@ func TestCheckRoot(t *testing.T) {
 	}{
 		{rspec.Spec{Windows: &rspec.Windows{HyperV: &rspec.WindowsHyperV{}}, Root: &rspec.Root{}}, "windows", specerror.RootOnHyperV},
 		{rspec.Spec{Windows: &rspec.Windows{HyperV: &rspec.WindowsHyperV{}}, Root: nil}, "windows", specerror.NonError},
+		{rspec.Spec{Windows: &rspec.Windows{}, Root: &rspec.Root{Path: filepath.Join(tmpBundle, "rootfs")}}, "windows", specerror.PathFormatOnWindows},
```

Comment:
Can we also check that [this][1] passes validation?  The regex is large, and a positive match would help convince me that it wasn't broken.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#example-windows

Reply:
updated.